tracing/tracing.go

```diff
@@ -25,30 +28,52 @@ func InitTracer(
 	endpoint string,
 	endpointType EndpointType,
 	samplingFraction float64,
-) (tp trace.TracerProvider, closer func(), err error) {
-	endpointOption := jaeger.WithAgentEndpoint(endpoint)
-	if endpointType == EndpointTypeCollector {
-		endpointOption = jaeger.WithCollectorEndpoint(endpoint)
+) (tp trace.TracerProvider, err error) {
+	tp = trace.NewNoopTracerProvider()
+
+	if endpoint == "" {
+		return tp, nil
+	}
+
+	host, port, err := net.SplitHostPort(endpoint)
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse host and port from tracing endpoint: %w", err)
 	}
 
-	tp, closer, err = jaeger.NewExportPipeline(
-		endpointOption,
-		jaeger.WithProcess(jaeger.Process{
-			ServiceName: serviceName,
-		}),
-		jaeger.WithSDK(&sdktrace.Config{DefaultSampler: sdktrace.ParentBased(sdktrace.TraceIDRatioBased(samplingFraction))}),
-		jaeger.WithDisabled(endpoint == ""),
+	endpointOption := jaeger.WithAgentEndpoint(
+		jaeger.WithAgentHost(host),
+		jaeger.WithAgentPort(port),
 	)
+
+	if endpointType == EndpointTypeCollector {
+		endpointOption = jaeger.WithCollectorEndpoint(jaeger.WithEndpoint(endpoint))
+	}
+
+	exp, err := jaeger.NewRawExporter(endpointOption)
 	if err != nil {
-		return tp, closer, fmt.Errorf("create jaeger export pipeline: %w", err)
+		return nil, fmt.Errorf("failed to create new raw tracing exporter: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("create new raw tracing exporter: %w", err)
```

We know we failed, because we wrap error? (: